src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -93,115 +96,136 @@ export class PropertyTemplateOutline extends Component {
+      let jsx = []
+      property.valueConstraint.valueTemplateRefs.map((rtId) => {
+        const resourceTemplate = _.find(this.state.nestedResourceTemplates, ['id', rtId])
+        jsx.push(<div className="row" key={shortid.generate()}>
+            <section className="col-sm-8">
+              <h5>{resourceTemplate.resourceLabel}</h5>
+            </section>
+            <section className="col-sm-4">
+              <PropertyActionButtons handleAddClick={this.handleAddClick}
+                                     handleMintUri={this.handleMintUri} key={shortid.generate()} />
+            </section>
+          </div>)
+        resourceTemplate.propertyTemplates.map((rtProperty) => {
+          let newReduxPath = Object.assign([], rtReduxPath)
+          newReduxPath.push(rtId)
+          newReduxPath.push(rtProperty.propertyURI)
+          const payload = { reduxPath: newReduxPath }
+          if (rtProperty.valueConstraint.defaults && rtProperty.valueConstraint.defaults.length > 0) {
+            payload['defaults'] = []
+            rtProperty.valueConstraint.defaults.map((row, i) => {
+              payload['defaults'].push({
+                id: i,
+                content: row.defaultLiteral,
+                uri: row.defaultURI
+              })
+            })
+          }
+          this.props.initNewResourceTemplate(payload)
+          jsx.push(<PropertyTemplateOutline key={shortid.generate()}
+                                            propertyTemplate={rtProperty}
+                                            reduxPath={newReduxPath}
+                                            initNewResourceTemplate={this.props.initNewResourceTemplate}
+                                            resourceTemplate={resourceTemplate} />)
+        })
+      })
+      return jsx
+    } else {
+      // TODO: remove this case
+      // if no valueTemplateRef, assume it is a confused propertyTemplate intended to be of type lookup
+      // NOTE: this case is due to orig LC resourceTemplates;  sinopia schemas v0.0.9 and up do NOT allow this
+      return <InputListLOC propertyTemplate = {property}
+                            lookupConfig = {getLookupConfigItem(property)}
+                            rtId = {property.rtId} />
+    }
   }
 
+  /**
+  * expanded vs collapsed property outline
+  * @private
+  */
   handleClick = (property) => (event) => {
     event.preventDefault()
     let newOutput = this.state.output
 
-    this.fulfillRTPromises(this.resourceTemplatePromises(property.valueConstraint.valueTemplateRefs)).then(() => {
-
-      const input = this.handleComponentCase(property)
-
-      let existingInput
-      newOutput.forEach((input) => {
-        if (this.props.propertyTemplate.propertyURI === input.props.propertyTemplate.propertyURI) {
-          existingInput = input
-          return
+    this.fulfillRTPromises(this.resourceTemplatePromises(property.valueConstraint.valueTemplateRefs))
+      .then(() => {
```

Comment:
I found the existing alignment a bit confusing, so I tried to make it more obvious what was going on with diff linebreaks, etc.